quixstreams/state/rocksdb/windowed/transaction.py

```diff
@@ -240,3 +218,46 @@ def get_windows(
                 result.append(((start, end), value))
 
         return result
+
+    def _set_latest_timestamp(self, prefix: bytes, timestamp_ms: int):
+        self._latest_timestamps[prefix] = timestamp_ms
+        self.set(
+            key=LATEST_TIMESTAMP_KEY,
+            value=timestamp_ms,
+            prefix=prefix,
+            cf_name=LATEST_TIMESTAMPS_CF_NAME,
+        )
+
+    def _get_last_expired_timestamp(self, prefix: bytes) -> Optional[int]:
```

Comment:
Get and set methods for latest timestamp and last expired are very similar. Did you think about some abstraction here?

Reply:
They are, but I don't think we need to abstract anything here. They are very simple